.ci/generate-buildkite-pipeline-premerge

```diff
@@ -188,6 +188,11 @@ function check-targets() {
     libclc)
       echo "check-all"
     ;;
+    mlir)
+      for t in check-mlir check-mlir-python; do
+        echo $t
```

Comment:
I'm a bit confused why `check-mlir-python` is necessary/useful on top of `check-mlir`? The intent is that if the Python tests are configured they should run with `check-mlir`.

Reply:
Sorry my bad, I see now from looking at the logs this runs the python tests twice. I've removed this now :+1:

Reply:
Thanks :)